js/src/admin/components/AdminNav.js

```diff
@@ -21,6 +21,17 @@ export default class AdminNav extends Component {
     );
   }
 
+  onupdate() {
+    if ($('.Dropdown-menu').children('.active').length > 0 && app.previous.type) {
+      $('#admin-navigation').animate(
+        {
+          scrollTop: $('.Dropdown-menu').children('.active')[0].offsetTop - 25,
```

Comment:
Perhaps instead of scrolling so the currently active one is at the top, it should scroll so it's in the middle of the screen? Also, I don't think it should scroll if it's already visible, only if it's not in the current viewport